### PR TITLE
(FBX-495) Add Windows ARM64 support

### PR DESCRIPTION
**Pull request by thomas-tu**
## Purpose of this PR:
Add Windows ARM64 support and update FBX SDK to 2020.3.4.

**JIRA ticket:**
[FBX-495](https://jira.unity3d.com/browse/FBX-495) Add support for Windows ARM64 platform
[FBX-501](https://jira.unity3d.com/browse/FBX-501) Update FBX SDK to 2020.3.4

**List of changes:**
- Add a python script `zip-fbxsdk.py` that can create fbxsdk zip archives for all platforms which will be uploaded to Stevedore
- Update CI and CMakeLists.txt to use fbxsdk 2020.3.4
- Build one single **universal** fbxsdk binding for Mac, instead of building for x86_64 and arm64 separately and then merging them into one file.
- Update build scripts to support building for x64 and ARM64 platforms based on the architecture of the build machine
- Also make changes to build scripts to better support building bindings using local fbxsdk in `fbxsdk_to_upload` folder before uploading to Stevedore
-  Upload a pre-built ARM64 Windows binding lib to `Editor/Plugins/WinARM64` folder (as a temp solution)
- Change Ubuntu image from Ubuntu-18 to Ubuntu-20 in CI

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.autodesk.fbx?pullRequest=411) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.autodesk.fbx?pullRequest=411) before we can accept your contribution.<br/><hr/>**Benoit Hudson** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.autodesk.fbx?pullRequest=411) it.</sub>

**Review by benoithudson — CHANGES_REQUESTED** [review]
We shouldn't include `proto.com.autodesk.fbx/Editor/Plugins/WinARM64/UnityFbxSdkNative.dll` in the commits.

File: .yamato/upload-fbxsdk-to-stevedore.yml [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
+# 4) On each platform:
+#    a) install FBX SDK using the installer
+#    b) pull the branch
+#    c) windows:
+#         py -3 .yamato/zip-fbxsdk.py "/c/Program Files/Autodesk/FBX/FBX SDK/2020.3.4"
+#         git commit fbxsdk_to_upload/fbxsdk-win-x64.7z
+#    d) mac:
+#         python .yamato/zip-fbxsdk.py "/Applications/Autodesk/FBX SDK/2020.3.4"
+#         git commit fbxsdk-mac-x64.7z
+#    e) linux: On linux, user needs to speficy where to install FBX SDK, for example at "/usr/FBX202034_FBXFILESDK_LINUX"
```
Comment (benoithudson) [inline .yamato/upload-fbxsdk-to-stevedore.yml:19]: *specify

Comment (mfe) [reply .yamato/upload-fbxsdk-to-stevedore.yml]: Fixed in ecbb0e31eb3bf033d9082ebdb8171fd834f029af

**Review by mfe — APPROVED** [review]
LGTM for now (binary will be properly pushed and pulled in a later version). 
Tested manually on WinAR64, CI is green, APV green too.

**Pull request merged**